.travis.yml

```diff
@@ -15,7 +15,7 @@ dist: xenial
 install:
     - pip install -U pip
     - pip install jaxlib
-    - pip install jax
+    - pip install -e git+https://github.com/google/jax.git@5861e37edd82ec5bd737fdb33b1ac6f2684e4827#egg=jax
```

Comment:
Are the changes here backwards compatible w.r.t. the latest JAX release? If not, we should probably also change `setup.py` so that users working on NumPyro's master branch are not left with a broken setup.

Reply:
It is not backward compatible, so I just update the setup file.